magefiles/magefile.go

```diff
@@ -499,6 +499,13 @@ func setRequiredEnvVars() error {
 			if !strings.Contains(jobName, "rehearse") {
 				os.Setenv(fmt.Sprintf("%s_CATALOG_URL", envVarPrefix), fmt.Sprintf("https://github.com/%s/%s", pr.RemoteName, pr.RepoName))
 				os.Setenv(fmt.Sprintf("%s_CATALOG_REVISION", envVarPrefix), pr.CommitSHA)
+				if isPRPairingRequired("release-service") {
+					os.Setenv(fmt.Sprintf("%s_IMAGE_REPO", envVarPrefix),
+						"quay.io/redhat-user-workloads/rhtap-release-2-tenant/release-service/release-service")
+					os.Setenv(fmt.Sprintf("%s_IMAGE_TAG", envVarPrefix), fmt.Sprintf("on-pr-%s", pr.CommitSHA))
```

Comment:
@psturc this is the commit for the catalog PR, not the release-service PR, right? So I should add some function like `isPRPairingRequired` that extracts that sha?

Reply:
Updated